Formula/dotnet.rb

```diff
@@ -51,6 +51,18 @@ def install
     (bin/"dotnet").write_env_script libexec/"dotnet", DOTNET_ROOT: libexec
   end
 
+  def caveats
+    <<~EOS
+      To ensure the Homebrew installed version of .NET Core is used, you have to
+      add the following line to your ~/.bash_profile, ~/.zshrc or
+      ~/.config/fish/config.fish file:
+
+        export DOTNET_ROOT="#{opt_libexec}"
+
+      Remember to source the file to update your current session.
```

Comment:
Sourcing your `.zshrc`, etc is not a great recommendation. See my other comment about this not being idempotent. It's not suggested in the caveats of keg-only formulae, so I don't think we should recommend this here either.

Reply:
I got the inspiration from the `autojump` formula:

https://github.com/Homebrew/homebrew-core/blob/0e914c10b6fd42cb0a028731eff41886569ae090/Formula/autojump.rb#L34-L35

Reply:
We should change that one too, as that recommendation can be harmful.

Reply:
Can we write something like this instead, perhaps?

> Remember to restart your session in order to make the `DOTNET_ROOT` variable available.